src/custom/pages/Claim/InvestmentFlow/InvestOption.tsx

```diff
@@ -183,16 +183,21 @@ export default function InvestOption({ approveData, updateInvestAmount, claim }:
         <InvestInput>
           <div>
             <span>
-              <b>Balance:</b>{' '}
+              <b>Balance:</b>
               <i>
-                {formatSmart(balance)} {currencyAmount?.currency?.symbol}
+                {formatSmart(balance) || 0} {currencyAmount?.currency?.symbol}
               </i>
               {/* Button should use the max possible amount the user can invest, considering their balance + max investment allowed */}
-              <button onClick={onMaxClick}>Invest max. possible</button>
+              <button onClick={onMaxClick}>(invest max. possible)</button>
             </span>
             <label>
+              <input
+                // disabled
```

Comment:
should it not be disabled via the attribute?

Reply:
My impression was this `disabled` attribute isn't set dynamically yet. And needed it to be not disabled for styling purposes. Commenting out as we need to address this still.

Reply:
got it, thought it was always disabled for this state, misread that. what are the criteria for disabled? we need to make an issue for this right?